screens/PracticalInfo/PracticalInfo.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from "react";
+import { Text, View, ScrollView } from "react-native";
+import styles from "./styles";
```

Comment:
Maybe this should be "Styles" instead?

Reply:
Which part do you want to be "Styles"? Is it import styles or is the file name that should be in uppercase?

Reply:
import Styles